docker-compose.yml

```diff
@@ -68,3 +71,4 @@ services:
 
 volumes:
   mysql: # eslint-disable-line yml/no-empty-mapping-value
+  files: # eslint-disable-line yml/no-empty-mapping-value
```

Comment:
This belongs to the above volume for sites/default/files, which is TBD.

For the `# eslint-disable-line yml/no-empty-mapping-value`, this seems to target eslint which we don't run anywhere, or do we?